packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -44,7 +44,7 @@ typedef ScrollableWidgetBuilder = Widget Function(
 ///
 /// The controller's methods cannot be used until after the controller has been
 /// passed into a [DraggableScrollableSheet] and the sheet has run initState.
-class DraggableScrollableController {
+class DraggableScrollableController extends ChangeNotifier {
```

Comment:
Please document what you can listen for.

Reply:
Added. Let me know if anything more is needed.